spec/file_list_spec.rb

```diff
@@ -53,7 +56,7 @@
 
   context 'when path directory contains current working dir and parent' do
     before(:each) do
-      allow(Dir).to receive(:foreach).and_yield('.').and_yield('.').and_yield('file1')
+      allow(Dir).to receive(:foreach).and_yield('.').and_yield('.').and_yield(file_1.name)
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.